src/replication.c

```diff
@@ -980,8 +968,9 @@ int startBgsaveForReplication(int mincapa, int req) {
     /* `SYNC` should have failed with error if we don't support socket and require a filter, assert this here */
     serverAssert(socket_target || !(req & SLAVE_REQ_RDB_MASK));
 
+    int slots_req = req & SLAVE_REQ_SLOTS_SNAPSHOT;
     serverLog(LL_NOTICE,"Starting BGSAVE for SYNC with target: %s%s",
-        socket_target ? "replicas sockets" : "disk",
+        socket_target ? (slots_req ? "import node" : "replicas sockets") : "disk",
```

Comment:
Instead of "import node", does it make sense to say something like "slot migration destination sockets". I feel like it may give a better idea to people who are investigating the logs if they do not know much about atomic slot migration. So, they won't be confused with some other tool may potentially import/export something.

Reply:
sounds good